### ISSUE TITLE
How to Specify Loss Function for Boosting Models in Darts? 

### ISSUE THREAD

**Issue by barmanroys** [unit8co/darts#2168]
How do I specify a custom loss function for models (especially, category boost) when using darts? In particular, I wish to use the [mean absolute percentage error ](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/mean_absolute_percentage_error) while fitting the category boost model, as shown below. 

```python3
    cat_booster = CatBoostModel(lags=LOOKBACK_WINDOW,
                                # How many lags of the target series
                                lags_past_covariates=LOOKBACK_WINDOW,
                                # How many of past
                                lags_future_covariates=[
                                    *range(1, 1 + HORIZON)],
                                # Future
                                output_chunk_length=HORIZON
                                )

    logging.info(
        msg=f'Fitting category boost with {len(target)} samples from {target.start_time()} till {target.end_time()}.')
    cat_booster.fit(series=target,
                    past_covariates=past_covariates,
                    future_covariates=future_covariates)
```

**Comment by dennisbader** [comment]
Hi @barmanroys, you can give CatBoost model parameters as keyword arguments at model creation (see below). It is also described in [our documentation](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.catboost_model.html#darts.models.forecasting.catboost_model.CatBoostModel) under model parameter `kwargs`.

```
CatBoostModel(..., loss_function="MAPE")
```

For our torch models (neural networks), you can use our builtin [MAPE loss](https://unit8co.github.io/darts/generated_api/darts.utils.losses.html#darts.utils.losses.MapeLoss):
You can find the `loss_fn` in the torch model documentation (e.g. for TiDEModel [here](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.tide_model.html#darts.models.forecasting.tide_model.TiDEModel))

```
from darts.utils.losses import MapeLoss
from darts.models import TiDEModel

TiDEModel(..., loss_fn=MapeLoss())
```

**Comment by barmanroys** [comment]
@dennisbader Thanks a lot for coming back. Yes, the train and inference worked with your suggestion for the CatBoost model. But if you do not mind two follow up questions 

- With my dataset, after I explicitly specify the loss function as MAPE, the MAPE on the validation dataset is bigger (i.e. worse). Is that possible? 
- Related, unless I specify the loss function, what's the default, which actually seems to give a smaller MAPE?
